config/dev.exs

```diff
@@ -138,3 +138,5 @@ end
 config :lightning, :is_resettable_demo, true
 
 config :lightning, :apollo, endpoint: "http://localhost:3000"
+
+config :lightning, :kafka_triggers, run_supervisors: true
```

Comment:
@jyeshe I initially added this so that I could stop the Kafka trigger supervisors running in the test env. I was going to ask if there is a more idiomatic way to do this, but for now I am going to use this config to apply a ENV-driven feature flag so that the initial deploy to prod can have Kafka trigger functionality disabled.